openpathsampling/tests/test_plumed_wrapper.py

```diff
@@ -0,0 +1,216 @@
+"""
+@author Alberto Perez de Alba Ortiz
+"""
+import openpathsampling as paths
+
+try:
+    import mdtraj as md
+except ImportError:
+    HAS_MDTRAJ = False
+else:
+    HAS_MDTRAJ = True
+
+try:
+    import openpathsampling.engines.openmm as peng_omm
+except ImportError:
+    HAS_OPENMM = False
+else:
+    HAS_OPENMM = True
+
+try:
+    import plumed
+except ImportError:
+    HAS_PLUMED = False
+else:
+    HAS_PLUMED = True
+    from openpathsampling.collectivevariables.plumed_wrapper\
+         import PLUMEDCV, PLUMEDInterface
+
+import pytest
+from .test_helpers import data_filename
+
+import numpy as np
+import os
+import copy
+
+
+class TestPLUMED(object):
+    def setup(self):
+        if not HAS_PLUMED:
+            pytest.skip("PLUMED module not installed.")
+        if not HAS_OPENMM:
+            pytest.skip("OpenMM module not installed.")
+        if not HAS_MDTRAJ:
+            pytest.skip("MDTraj module not installed.")
+        self.topology = peng_omm.tools.topology_from_pdb(data_filename(
+                                                         "plumed_wrapper/" +
+                                                         "AD_initial_frame" +
+                                                         ".pdb"))
```

Comment:
Indeed the `self.topology = self.trajectory[0].topology` works. However, back in the early stages I decided to first give the topology because it is used to initialize the PLUMED object (number of atoms and molinfo). I thought it wouldn't be so nice to initialize already with a specific trajectory. I preferred to initialize an object with the topology and then to calculate CVs of different trajectories (of the same system) using the same object. 

It boils down to the fact that molinfo cannot be set after a CV has been created.